dhall-toml/src/Dhall/DhallToToml.hs

```diff
@@ -0,0 +1,348 @@
+        \                                                                                \n\
+        \Explanation: The tomland library cannot handle records in nested arrays. You    \n\
+        \can check the status of this feature at:                                        \n\
+        \   https://github.com/kowainik/tomland/issues/385                               \n\
+        \                                                                                \n\
+        \For example:                                                                    \n\
+        \    ┌─────────────────────────┐                                                 \n\
+        \    | { x = [[ { a = 1 } ]] } |                                                 \n\
+        \    └─────────────────────────┘                                                 \n\
+        \                                                                                \n\
+        \" <> insert e
+
+    show (NotARecord e) =
+        _ERROR <> ": The root object converted to TOML must be a record                  \n\
+        \                                                                                \n\
+        \Explanation: A TOML file must represent a table, so primitive values and        \n\
+        \❰List❱s cannot be converted by themselves. Consider nesting the value in a      \n\
+        \record with arbitrary fields.                                                   \n\
+        \                                                                                \n\
+        \For example, from:                                                              \n\
+        \    ┌────┐                                                                      \n\
+        \    | 42 |                                                                      \n\
+        \    └────┘                                                                      \n\
+        \into                                                                            \n\
+        \    ┌────────────────────────┐                                                  \n\
+        \    | { meaningOfLife = 42 } |                                                  \n\
+        \    └────────────────────────┘                                                  \n\
+        \                                                                                \n\
+        \" <> insert e
+
+    show (HeterogeneousArray e) =
+        _ERROR <> ": Heterogeneous arrays are not currently supported                    \n\
+        \                                                                                \n\
+        \Explanation: The tomland library cannot handle records in nested arrays. You    \n\
+        \can check the status of this feature at:                                        \n\
+        \   https://github.com/kowainik/tomland/issues/373                               \n\
```

Comment:
This seems to be the same explanation as for the `UnsupportedArray` error?!

Reply:
I'm still surprised that this explanation has the same text as the one for `UnsupportedArray` but links to a different issue. Could you check?